modules/_labs/tabs/react/lib/TabList.tsx

```diff
@@ -20,58 +29,52 @@ const TabsListContainer = styled('div')({
   borderBottom: `1px solid ${commonColors.divider}`,
 });
 
-const TabsListInnerContainer = styled('div')({
+const TabsListInnerContainer = styled('div')<StyledType>({
   display: `flex`,
   margin: `0 ${spacing.m}`,
 });
 
-const TabList = ({children, ...elemProps}: TabListProps) => {
-  const tabsListRef = React.useRef<HTMLDivElement>(null);
-  // const [tabIndicatorRef, setDimensions] = useIndicator(tabsListRef);
-  const {setIntentTab, setActiveTab, intentTab, programmaticFocusRef} = useTab();
-
-  const onKeyDown = (event: React.KeyboardEvent) => {
-    switch (event.key) {
-      case 'ArrowLeft':
-      case 'Left':
-        setIntentTab('previous');
-        break;
-      case 'ArrowRight':
-      case 'Right':
-        setIntentTab('next');
-        break;
-      case 'Home':
-        setIntentTab('first');
-        break;
-      case 'End':
-        setIntentTab('last');
-        break;
-      case 'Enter':
-      case ' ':
-        setActiveTab(intentTab);
-        event.preventDefault(); // prevent clicking this button
-        break;
-      default:
-        break;
-    }
-  };
-
-  const resetProgrammaticFocus = () => {
-    programmaticFocusRef.current = false;
-  };
-
-  return (
-    <TabsListContainer ref={tabsListRef}>
-      <TabsListInnerContainer
-        role="tablist"
-        onKeyDown={onKeyDown}
-        onFocus={resetProgrammaticFocus}
-        {...elemProps}
-      >
-        {children}
-      </TabsListInnerContainer>
-    </TabsListContainer>
+/**
+ * Reset the cursor to the active tab when the tab list looses focus
+ */
+const useResetCursorOnBlur = ({state, events}: TabsModel, elemProps = {}) => {
```

Comment:
I'm not 100% this is a sharable hook. It depends on the `state.activeTab` which is specific to the Tabs model. If we change `activeTab` to be `activeId` or `activeCursor`, it might apply to something like a Select as well which also resets cursors when focus is lost.